### DISCUSSION TITLE
Commit Name Convention (add prefix)

### DISCUSSION THREAD

**Discussion by housengw** [lf-lang/lingua-franca#797] [Ideas]
Should we add a prefix to our commit messages to specify the related topic of the commit?

For example, we could do [Cpp] for commits related to the Cpp target and [Federated][Cpp] for commits related to the federated execution of the Cpp target.

**Comment by cmnrd** [comment]
I think that is a good idea. I was using "cpp:" for C++ related commits (but not consistently). However, prefixes should be short as the length of the first line in a git commit is typically limited. I think something like [Federated][Cpp] would be too verbose.
